src/modules/root-container/utils/focus-within.ts

```diff
@@ -1,24 +1,48 @@
 // inspired by https://gist.github.com/aFarkas/a7e0d85450f323d5e164
-const focusWithin = () => {
-  const slice = [].slice;
-  const removeClass = (elem: any) => {
-    elem.classList.remove('focus-within');
+const focusWithin = (
+  rootElement: Element,
+  onFocusEnter: () => void,
+  onFocusLeave: () => void,
+) => {
+  const clearClass = () => {
```

Comment:
seems like `clearClass` could be extracted from `focusWithin` fn. To not create it every time

Reply:
smth like `removeFocusClass(rootElement)` & `addFocusClass(rootElement, activeElement)`

Reply:
You don't really need utils for focusWithin anywhere else, since it's more like polyfill for [focus-within](https://caniuse.com/#feat=css-focus-within)

Reply:
Cool. One more change, could make me heapy. If U move `focus-within` class name to constant